en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md

```diff
@@ -0,0 +1,224 @@
+
+### In deployment.toml
+
+You can include encrypted passwords in the `deployment.toml` file found in the `<IS_HOME>/repository/conf` directory by using the `$secret{alias}` format as shown below:
+
+!!! note
+    You may pass passwords as environment variables and system properties instead of directly setting the values. See [Set Passwords using Environment Variables/System Properties]({{base_path}}/deploy/security/set-passwords-using-environment-variables-or-system-properties) for instructions.
+
+```toml
+[super_admin]
+username="admin"
+password="$secret{admin_password}"
+
+[keystore.tls]
+password = "$secret{keystore_password}" 
+alias = "$secret{keystore_password}" 
+key_password = "$secret{key_password}"  
+
+[truststore]                  
+password = "$secret{keystore_password}" 
+```
+
+### In log4j2.properties
+
+You may include encrypted passwords in the `log4j2.properties` file found in the `<IS_HOME>/repository/conf` directory. For example, you may set `log4j.appender.LOGEVENT.password` as shown below:
+
+```
+log4j.appender.LOGEVENT.password=secretAlias:log4j2_password
+```
+
+## Change encrypted passwords
+
+You may follow the steps below to change passwords that are already encrypted.
+
+1. Shut down the {{product_name}}.
+
+2. On a command prompt, navigate to the `<IS_HOME>/bin/` directory where the cipher tool scripts reside.
+
+{% if product_name == "WSO2 Identity Server" and is_version != "7.0.0" %}
+
+3. Run the cipher tool by executing the command corresponding to your encryption method and operating system:
+
+    === "Symmetric encryption"
+
+        - **For Linux**: `./ciphertool.sh -Dconfigure -Dsymmetric`
+
+        - **For Windows**: `ciphertool.bat -Dconfigure -Dsymmetric`
+
+    === "Asymmetric encryption"
+
+        - **For Linux**: `./ciphertool.sh -Dconfigure`
+
+        - **For Windows**: `ciphertool.bat -Dconfigure`
+
+{% else %}
+
+3. Run the cipher tool by executing the command corresponding to your operating system:
+
+    * Linux: `./ciphertool.sh -Dchange`
+    * Windows: `ciphertool.bat -Dchange`
+
+{% endif %}
+
+4. You will be prompted for the primary keystore password. Enter the password. For the default keystore, it is `wso2carbon`.
+
+5. A numbered list of all encrypted password aliases will be displayed. Enter the number corresponding to the alias of the password you want to change.
+
+6. The system will then prompt you (twice) to enter the new password. Enter your new password.
+
+{% if product_name == "WSO2 Identity Server" and is_version != "7.0..0" %}
+## Rotating Encryption Secrets
+
+!!! note
+    To support symmetric encryption, you must specify a PKCS12 type keystore as the internal keystore.
+
+You can rotate encryption keys by switching between symmetric and asymmetric encryption or by changing the encryption keys within the same encryption mode.
+
+1. For **symmetric** encryption, use the command below to add the new key to the existing keystore with a new alias. 
```

Comment:
Shall we use the === "Symmetric encryption" sections here too, for consistency?

Reply:
fixed with https://github.com/hwupathum/docs-is/pull/5/commits/3df8725bb45a9cfc4c10b1ae1c1237de5715212f